README.md

```diff
@@ -107,6 +109,73 @@ This example shows how to use the Splunk Enterprise SDK for JavaScript and Node.
         });
     });
 ```
+##### Login with sessionKey
+
+```shell
+# Create a sessionKey
+curl -k -u <username>:<password>  <scheme>://<host>:<port>/services/auth/login -d username=<username> -d password=<password>
+```
+
+```javascript
+var serviceWithSessionKey = new splunkjs.Service(
+    {
+        // Replace the host if you are accessing remote host
+        scheme: 'https',
+        host: 'localhost',
+        port: '8089',
+        sessionKey: SESSION_KEY, // Add your sessionKey here
+        version: '8',
+    });
+
+serviceWithSessionKey.get("search/jobs", { count: 1 }, function (err, res) {
+    if (err) {
+        console.log(err);
+    } else }
+        console.log("Login successful with sessionKey");
+    }
+});
+```
+
+##### Login with token
+
+```shell
+#### From shell ####
+# Enable token authetication
+curl -k -u <username>:<password> -X POST <scheme>://<host>:<port>/services/admin/token-auth/tokens_auth -d disabled=false
+
+# Create a token
+curl -k -u <username>:<password> -X POST <scheme>://<host>:<port>/services/authorization/tokens?output_mode=json --data name=admin --data audience=Users --data-urlencode expires_on=+30d
```

Comment:
Here I think `name=admin` could refer to any valid username so suggest changing to `name=<username>` - I just tried this so I think that should be correct